findingcheapestgpus.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Finding the Cheapest GPUs'
+---
+Shadeform makes it easy to find and deploy VM and containers to the cheapest GPU instances across many different clouds. Let's show how we can do this with the UI and the API, specifically for an instance with a single 80 GB A100 GPU.
+
+
+## Using the Shadeform UI
+
+### 1. Searching for GPU's
+Once you've created an account and added funds to your [Wallet](https://platform.shadeform.ai/settings/billing) then you can see available instances on the homepage. 
+
+![GPU Filter](images/1x80GBA100.png)
+
+### 2. Launching a GPU
+We can select for a single 80 GB A100 as shown. Listed are the instances that are currently available. We can select to launch one.
+![Launching](images/launchgpu.png)
+### 3. Waiting for startup
+When we go into the Running instances [tab](https://platform.shadeform.ai/running), we can see that this VM is pending while it spins up.
+![Pending](images/pending.png)
+### 4. Scheduling an instance
+If we want an even cheaper instance but are willing to wait, we can also Schedule and instance instead. 
+![Schedule](images/schedule.png)
+Shadeform will handle acquiring the instances as soon as they come available.
+
+### 5. Waiting for scheduling
+Once Scheduled, you can see your scheduled requests under [Scheduled Instances](https://platform.shadeform.ai/reserve)
+![ManageScheduled](images/managescheduled.png)
+ 
+## Using the Shadeform API
+Once you've created an account and added funds to your [Wallet](https://platform.shadeform.ai/settings/billing), then you can get an API Key by going to the [API Keys page](https://platform.shadeform.ai/settings/api). Click on “Generate” to create a new API key.
+
+We've created a [python notebook](Scriptlinkhere.com) that uses the Shadeform API to do this for you. It's light in dependencies so it can run almost anywhere. Let's go through it:
```

Comment:
We'll have to update the link before deploying the changes. What is the best way of hosting a python notebook script?

Reply:
I think that a github repo is best. There you put a lot of different things. You could call it "examples", "cookbook", or something like that

Reply:
Cool, I created a repo here: https://github.com/shadeform/examples add you as a contributor

Reply:
I cleaned up the notebook, cloned the repo, and am trying to push to a branch for a PR in that repo. It's giving me a 403, do I have write access? 
My error message:
`remote: Permission to shadeform/examples.git denied to aowen14.
fatal: unable to access 'https://github.com/shadeform/examples.git/': The requested URL returned error: 403`